src/DockManager.cpp

```diff
@@ -88,7 +88,7 @@ enum eStateFileVersion
 	CurrentVersion = Version1//!< CurrentVersion
 };
 
-static CDockManager::ConfigFlags StaticConfigFlags = CDockManager::DefaultNonOpaqueConfig;
+static CDockManager::m_ConfigFlags StaticConfigFlags = CDockManager::DefaultNonOpaqueConfig;
```

Comment:
Class definitions should not have the `m_` prefix.

Reply:
Not a problem - what would you suggest to avoid the ConfigFlags conflict there?

Reply:
Where is the conflict?